cmd/kubectl/kubectlcobra/inventory.go

```diff
@@ -81,3 +82,99 @@ func (i *Inventory) String() string {
 		i.GroupKind.Group, fieldSeparator,
 		i.GroupKind.Kind)
 }
+
+// InventorySet encapsulates a grouping of unique Inventory
+// structs. Organizes the Inventory structs with a map,
+// which ensures there are no duplicates. Allows set
+// operations such as merging sets and subtracting sets.
+type InventorySet struct {
+	set map[string]*Inventory
```

Comment:
Is there any particular reason why we are using pointer to Inventory here? Since Inventory just capture a few strings and the GroupKind, it seems like this could just be `map[string]Inventory`.